docs/source/guides/transcoder_benchmark.rst

```diff
@@ -0,0 +1,135 @@
+How To Benchmark A Transcoder
```

Comment:
I think "How To Benchmark Transcoding" might be a more descriptive title since this guide can be used to benchmark transcoding regardless of whether you identify as an orchestrator or transcoder.